core/package.json

```diff
@@ -46,7 +46,7 @@
     "@stencil/vue-output-target": "^0.6.0",
     "@types/jest": "^26.0.20",
     "@types/node": "^14.6.0",
-    "@types/puppeteer": "5.4.3",
+    "@types/puppeteer": "^5.4.4",
```

Comment:
As of v6 (IIRC, but it may be later), Puppeteer includes their own typings, we may be able `npm uninstall` these from the community